docs-src/manual/cli.rst

```diff
@@ -0,0 +1,18 @@
+Command-line interface
+======================
+
+The ``hypothesis fuzz`` command is the main entrypoint to HypoFuzz. Installing HypoFuzz automatically adds the ``fuzz`` sub-command to the existing :ref:`Hypothesis CLI <hypothesis:hypothesis-cli>`.
+
+``hypothesis fuzz``
+-------------------
+
+.. command-output:: hypothesis fuzz --help
```

Comment:
New problem, it's not rendering: https://hypofuzz.com/preview-102/example-dashboard/docs/manual/cli.html#hypothesis-fuzz

Reply:
oh no... `hypothesis fuzz --help` requires hypofuzz to be installed, but building docs on sdist `pip install` means hypofuzz isn't installed yet.

Reply:
Time to literalinclude a text file then, and have a test that the file is up to date.